test_framework/Jenkinsfile

```diff
@@ -6,6 +6,7 @@ def BUILD_TRIGGER_BY = "\n${currentBuild.getBuildCauses()[0].shortDescription}"
 def SELINUX_MODE = params.SELINUX_MODE ? "SELINUX_MODE=${params.SELINUX_MODE}" : ""
 
 def CREDS_ID = JOB_BASE_NAME == "longhorn-tests-regression" ? "AWS_CREDS_RANCHER_QA" : "AWS_CREDS"
+def REGISTRATION_CODE_ID = params.ARCH == "amd64" ? "REGISTRATION_CODE" : "REGISTRATION_CODE_ARM64"
```

Comment:
What reason we need to difference the value based on arch?

Reply:
The code is generated per OS version and arch in SUSE customer center. If we upgrade to sle-micro 5.4 in the future, new codes need to be generated for sle-micro 5.4 too.